kiro-synth-host/src/midi/drivers/jackmidi.rs

```diff
@@ -0,0 +1,80 @@
+use anyhow::Result;
+use thiserror::Error;
+
+use kiro_midi_core::decoder::{CallbackResult, Decoder, DecoderCallbacks};
+use kiro_midi_core::messages::Message;
+
+use crate::midi::drivers::{MidiError, MidiHandler};
+
+use jack::{Client, Control, ProcessHandler, ProcessScope};
```

Comment:
Please move this line to the top (after `use thiserror::Error`)
I usually put all the external imports first, then the project ones.
I don't know if `cargo fmt` can be configured to do that kind of things automatically.
Thanks.